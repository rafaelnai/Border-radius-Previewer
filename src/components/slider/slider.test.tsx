import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Slider from "./slider";
import "../../../config";

describe("Slider component", () => {
  it("to match snapshot", () => {
    const wrapper = shallow(<Slider />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("to match snapshot", () => {
    const wrapper = shallow(<Slider vertical />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
