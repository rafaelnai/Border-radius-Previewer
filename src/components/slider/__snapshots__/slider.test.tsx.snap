// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider component to match snapshot 1`] = `
<input
  className="slider-0-2-1 slider-d0-0-2-2"
  max="100"
  min="1"
  name="simple-slider"
  title="Simple slider"
  type="range"
/>
`;

exports[`Slider component to match snapshot 2`] = `
<input
  className="slider-0-2-1 slider-d1-0-2-3"
  max="100"
  min="1"
  name="simple-slider"
  title="Simple slider"
  type="range"
/>
`;
